packages/windows/install-mroonga.ps1

```diff
@@ -0,0 +1,40 @@
+
+Param(
+  [Parameter(mandatory=$true)][String]$mariadbVersion,
+  [Parameter(mandatory=$true)][String[]]$platforms
+)
+
+function Wait-UntilRunning($cmdName) {
+  do
+  {
+    $Running = Get-Process $cmdName -ErrorAction SilentlyContinue
+    Start-Sleep -m 500
+  } while (!$Running)
+}
+
+function Wait-UntilTerminate($cmdName) {
+  do
+  {
+    $Running = Get-Process $cmdName -ErrorAction SilentlyContinue
+    Start-Sleep -m 500
+  } while ($Running)
+}
+
+function Install-Mroonga($mariadbVer, $arch, $installSqlDir) {
+  cd "mariadb-$mariadbVer-$arch"
+  Start-Process .\bin\mysqld.exe
+  Wait-UntilRunning mysqld
+  Get-Content "$installSqlDir\install.sql" | .\bin\mysql.exe -uroot
+  Start-Sleep -m 1000
+  Start-Process .\bin\mysqladmin.exe -ArgumentList "-uroot shutdown"
+  Wait-UntilTerminate mysqld
+  cd ..
+}
+
+$installSqlDir = ".\share\mroonga"
+
+foreach ($arch in $platforms)

```

Comment:
Do we need to support multiple platforms?

Reply:
No. fixed it.